frontend/src/app/shared/shared.module.ts

```diff
@@ -26,7 +28,8 @@ import { PrimeComponentsModule } from './modules/prime-components/prime-componen
         ReactiveFormsModule,
         LoadingSpinnerComponent,
         NotFoundComponent,
-        PrimeComponentsModule
+        PrimeComponentsModule,
+        LogoComponent
```

Comment:
Why do you add this here?

Reply:
I think logo will be used in other modules, so I've decided to add logo to shared

Reply:
I'm not sure about it, so for now it can be in the registration module

Reply:
Ok, I'll move it to registration module